packages/pwa-kit-create-app/package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "commander": "^2.20.0",
     "deepmerge": "2.2.1",
-    "fs-extra": "7.0.1",
```

Comment:
This is a major bump, it could be a breaking change

Reply:
I went through the [changelog](https://github.com/jprichardson/node-fs-extra/blob/master/CHANGELOG.md), it doesn't look like there's anything that would impact us.